Add new benchmark: Portuguese bench

zxcvuser opened the pull request:
PortugueseBench is a benchmark for tasks in European Portuguese that cover several evaluation areas. The datasets consist of professional translations of relevant English datasets and newly created datasets in European Portuguese. The README.md contains detailed information on all the tasks included in the benchmark.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2156) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2156) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/EleutherAI/lm-evaluation-harness?pullRequest=2156) it.</sub>

baberabb commented:
Thanks for this PR. This one looks good except the minor issue I identified. Could you also run
```
pre-commit run --all-files
```
for some minor linting issues.

zxcvuser commented:
These are the changes done:
- Added the benchmark info in lm_eval/tasks/README.md
- Replaced "-" by "_" in the create_files script in flores_pt and added `weight_by_size: false`
- Run linters
With these, it should all be fine now. Thank you!

haileyschoelkopf commented on lm_eval/tasks/portuguese_bench/flores_pt/_flores_common_yaml:
```diff
@@ -0,0 +1,28 @@
+group: flores
```
discussed in https://discord.com/channels/729741769192767510/1285953521589686323 !

haileyschoelkopf reviewed (approved):
Thanks for the PR and for going back and forth with us on this!
